src/MonologCreator/Handler/Udp.php

```diff
@@ -12,52 +12,29 @@
  */
 class Udp extends \Monolog\Handler\AbstractProcessingHandler
 {
-    /**
-     * @var \Monolog\Handler\SyslogUdp\UdpSocket
-     */
-    private $_socket = null;
-
-    /**
-     * @param \Monolog\Handler\SyslogUdp\UdpSocket $socket
-     * @param bool|int                             $level
-     * @param bool                                 $bubble
-     */
     public function __construct(
-        \Monolog\Handler\SyslogUdp\UdpSocket $socket,
-        $level = \Monolog\Logger::DEBUG,
-        $bubble = true
+        private \Monolog\Handler\SyslogUdp\UdpSocket $socket,
+        int|string|\Monolog\Level $level = \Monolog\Level::DEBUG,
```

Comment:
Why can this level be so many different types?

Reply:
I've have no idea, I've used the same parameter types as used in Monolog itself. see https://github.com/Seldaek/monolog/blob/main/src/Monolog/Handler/AbstractHandler.php#L36

Reply:
Odd, but if they say so then who are we to argue ;)